_faq/bugs-play-downloaded-videos.html

```diff
@@ -0,0 +1,18 @@
+---
+categories:
+- player
+- bugs
+type: info
+title: "Play downloaded videos in-app"
+---
+<section>
+    <p>In order to play downloaded videos, you need to use an external video player.</p>
+    <p>There have been considerations implementing this directly into NewPipe. This idea was dropped due to these reasons:</p>
```

Comment:
```suggestion
    <p>The team has considered implementing this directly into NewPipe. This idea was dropped due to these reasons:</p>
```

Reply:
I would leave that as it is currently.

Reply:
I dont mind either.

Reply:
You can change it to something else if you like, but the current phrasing is grammatically off.

Reply:
> There have been considerations implementing this directly into NewPipe.

I don't see a problem in here.

Reply:
The line reads like two disjointed phrases instead of one long sentence. You need a word in between to join the two, like "considerations around implementing", or "considerations regarding implementing". But we're making it more and more complicated by adding words. What I suggested above is simpler and easier to read and understand.